### PR TITLE
:tada: SetUp

### PR DESCRIPTION
**Pull request by alicespringfeld**
removed starter code of vite-boilerplate

### CODE REVIEW

**Review by davidwobido — APPROVED** [review]
lgtm 💥

**Review by JulianSchumacher99 — APPROVED** [review]
lgtm 🙂

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/app/favicon.svg" />
```
Comment (lmachens) [inline index.html:5]: Make sure to remove reference to `favicon.svg` in your `index.html`, otherwise, you will get an error

**Review by lmachens — CHANGES_REQUESTED** [review]

**Review by Ziedlassoued — APPROVED** [review]
Great job 👍

**Pull request merged**